### DISCUSSION TITLE
Push command to another plugin

### DISCUSSION THREAD

**Discussion by iBaird44** [kuestess/homebridge-platform-insteonlocal#296] [General]
Currently I can set up an automation from HomeKit to activate a second non insteon light when I control one of my insteon devices HOWEVER if that same command is done from the physical switch obviously the second device doesn’t get the command as HomeKit has been bypassed. I do see the insteon command show up in the Homebridge status though so is it possible to build an automation from within Homebridge that will send the same automation command that HomeKit does?  In my case if Homebridge see’s an Insteon command to then send a Tuya command as well. I’ve searched to an avail but perhaps I’m not looking in the right place.

Iain

**Comment by ctgreybeard** [comment]
Yes, you can do that. Use an Automation. One of the auxiliary Homekit apps are better than the Home app for Automations.

Use as a trigger the Insteon changing state to On and the Actionto set a Scene to turn On the other light. Then create the opposite.

I have a s8milar automationto turn my driveway light on if either garage door opens or closes.

**Comment by iBaird44** [comment]
For whatever reason the polling for Insteon in my case isn’t always accurate or on time and the message doesn’t always get through for a minute or so if I use automations from HomeKit. I think I have some noisy Insteon devices that bung up the transmissions. However if the automation was from within Homebridge then the delay getting to HomeKit wouldn’t matter.

**Reply by ctgreybeard** [reply]
You should have the pairing set up both directions. I don’t myself so I am not one to criticize. If it is set up properly then the action goes directly to Homebridge. But I understand.
